share/spice/biblio_search/books_by_author/biblio_search_books_by_author.handlebars

```diff
@@ -0,0 +1,4 @@
+<span>{{name}}</span>
```

Comment:
Yes I will remove all the unnecessary handlebars. I just wanted to see how one works.  After restructuring package hopefully it will work.